### PR TITLE
Make the test clearer that discovery alg is ignored

### PR DESCRIPTION
**Pull request by adamjmcgrath**
### Description

Update test to make it clearer that if the discovery doc is compromised the `idTokenSigningAlg` can not be overridden from the client configuration options.

And add a test to show "none" is not allowed for `idTokenSigningAlg` config

### CODE REVIEW

File: test/config.tests.js [hunk 1]
```diff
@@ -370,6 +370,18 @@ describe('get config', () => {
     );
   });
 
+  it('should not allow "none" for idTokenSigningAlg', () => {
```
Comment (lbalmaceda) [inline test/config.tests.js:373]: the test above is for the alg received from the .well-known and this one is for the configuration/options passed by the user. Is that correct? If so, looks good, and thanks for updating these.

Comment (adamjmcgrath) [reply test/config.tests.js:373]: yep - I thought I'd add one for the configuration as well
